### ISSUE TITLE
Pay attention to depreciation / vulnerability warnings

### ISSUE THREAD

**Issue by JRJurman** [Tram-One/tram-one#73]
## Summary
On builds sometimes we get depreciation warnings. Github at times also announces vulnerability warnings. While many of these are for the build steps and almost all of them are for external dependencies, if nothing else, we should look into what it might take to update these dependencies (even if it means making PRs on those repos).

## Checklist
Here is a checklist of the warnings we get:
_to be filled in_

**Comment by JRJurman** [comment]
```
npm WARN deprecated object-keys@0.2.0: Please update to the latest object-keys
```

**Comment by JRJurman** [comment]
I'm gonna treat the `object-keys` as it's strictly a dependency of our build tools. The effected branches are:
```
├─┬ rollup-plugin-node-builtins@2.1.2
│ ├─┬ browserify-fs@1.0.0
│ │ ├─┬ level-filesystem@1.2.0
│ │ │ ├── concat-stream@1.6.0 deduped
│ │ │ ├── errno@0.1.7 deduped
│ │ │ ├─┬ fwd-stream@1.0.4
│ │ │ │ └─┬ readable-stream@1.0.34
│ │ │ │   ├── core-util-is@1.0.2 deduped
│ │ │ │   ├── inherits@2.0.3 deduped
│ │ │ │   ├── isarray@0.0.1
│ │ │ │   └── string_decoder@0.10.31
│ │ │ ├─┬ level-blobs@0.1.7
│ │ │ │ ├── level-peek@1.0.6 deduped
│ │ │ │ ├── once@1.4.0 deduped
│ │ │ │ └─┬ readable-stream@1.1.14
│ │ │ │   ├── core-util-is@1.0.2 deduped
│ │ │ │   ├── inherits@2.0.3 deduped
│ │ │ │   ├── isarray@0.0.1
│ │ │ │   └── string_decoder@0.10.31
│ │ │ ├─┬ level-peek@1.0.6
│ │ │ │ └── level-fix-range@1.0.2
│ │ │ ├─┬ level-sublevel@5.2.3
│ │ │ │ ├─┬ level-fix-range@2.0.0
│ │ │ │ │ └── clone@0.1.19
│ │ │ │ ├─┬ level-hooks@4.5.0
│ │ │ │ │ └── string-range@1.2.2 deduped
│ │ │ │ ├── string-range@1.2.2
│ │ │ │ └─┬ xtend@2.0.6
│ │ │ │   ├── is-object@0.1.2
│ │ │ │   └─┬ object-keys@0.2.0
│ │ │ │     ├── foreach@2.0.5 deduped
│ │ │ │     ├── indexof@0.0.1 deduped
│ │ │ │     └── is@0.2.7
│ │ │ ├── octal@1.0.0
│ │ │ ├── once@1.4.0 deduped
│ │ │ └── xtend@2.2.0
│ │ ├─┬ level-js@2.2.4
│ │ │ ├─┬ abstract-leveldown@0.12.4
│ │ │ │ └── xtend@3.0.0
│ │ │ ├── idb-wrapper@1.7.2
│ │ │ ├── isbuffer@0.0.0
│ │ │ ├── ltgt@2.2.0
│ │ │ ├── typedarray-to-buffer@1.0.4
│ │ │ └─┬ xtend@2.1.2
│ │ │   └── object-keys@0.4.0
```
```
├─┬ webpack-dev-server@2.11.1
│ ├── ansi-html@0.0.7
│ ├─┬ array-includes@3.0.3
│ │ ├─┬ define-properties@1.1.2
│ │ │ ├── foreach@2.0.5
│ │ │ └── object-keys@1.0.11
```
